src/status_im/contexts/preview/feature_flags/view.cljs

```diff
@@ -0,0 +1,30 @@
+(ns status-im.contexts.preview.feature-flags.view
```

Comment:
we need to rename `preview` to something like `developer` or something to that effect

Reply:
Or just drop it all together? 
`feature-flags` in itself is a good context name.

Reply:
ah, I believe there is some aim to not overload the context folder with too many direct folders as these are mainly higher concepts (related to designs files/functionality) mostly.

e.g 
contexts/chat
contexts/shell
contexts/communities

and the problem becomes that the codebase is a bit more confusing to navigate if we start adding small sets of functionality within contexts.

but maybe feature flags can live somewhere better anyway 🤔 
I'll leave for now but happy to have it moved going forward